cmd/kubeadm/app/constants/constants_windows.go

```diff
@@ -1,7 +1,7 @@
 // +build windows
 
 /*
-Copyright 2019 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```

Comment:
here as well.

Reply:
whoops sorry. Somehow thought I split those up in this PR.